### ISSUE TITLE
Feature request: Mute, with display on toolbar

### ISSUE THREAD

**Issue by Grazfather** [talonvoice/talon#27]
It'd be nice to be able to mute talon, for privacy/ease of mind reasons, with it indicated on the menubar icon.

**Comment by lunixbochs** [comment]
```
from talon import app, voice
from talon.api import lib

def on_menu(item):
    if item == '!Enable Speech Recognition':
        if voice.talon.enabled:
            voice.talon.disable()
        else:
            voice.talon.enable()
            voice.talon.enabled = True
        lib.menu_check(b'!Enable Speech Recognition', voice.talon.enabled)

app.register('menu', on_menu)
lib.menu_check(b'!Enable Speech Recognition', voice.talon.enabled)
```

**Comment by Grazfather** [comment]
That's great :)

Now a way to change the icon to know at a glance whether it's enabled or not?

**Comment by lunixbochs** [comment]
```from talon import app, voice
from talon.api import lib

def set_enabled(enable):
    if enable:
        voice.talon.enable()
        app.icon_color(0, 0.7, 0, 1)
    else:
        voice.talon.disable()
        app.icon_color(1, 0, 0, 1)
    lib.menu_check(b'!Enable Speech Recognition', enable)

def on_menu(item):
    if item == '!Enable Speech Recognition':
        set_enabled(not voice.talon.enabled)

app.register('menu', on_menu)
set_enabled(voice.talon.enabled)

**Comment by lunixbochs** [comment]
Something like this will likely ship with Talon soon, so I'll leave this issue open until then.

**Comment by lunixbochs** [comment]
Included in 0.0.8
